integration/devmode/devmode_suite_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package devmode
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/hyperledger/fabric/integration/nwo"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestEndToEnd(t *testing.T) {
```

Comment:
Why is this EndToEnd instead of DevMode?

Reply:
fixed.